src/components/shared/layout/toggleHeader.vue

```diff
@@ -0,0 +1,89 @@
+<template>
```

Comment:
Can we bring this filename's casing inline with the rest of the components in the styleguide?

Reply:
Ah my bad, thats updated to Pascal case now